docs/BuildFromSource.md

```diff
@@ -392,3 +398,33 @@ When attempting to restore servicing tags e.g. `v3.1.7`,  the NuGet.config file
 ```
 
 The `darc-int-...` feeds in NuGet.config are used only when building internally and are not needed after the tags are created. Delete all such entries in the file and retry.
+
+### Common error: error MSB4019: The imported project "&hellip;\artifacts\bin\GenerateFiles\Directory.Build.props" was not found
```

Comment:
Wondering whether it makes sense to delete the entire **Troubleshooting** section or reduce it down to a pointer to the BuildErrors.md document. Think I should either duplicate these errors in the two files or do that. Thoughts❔

Reply:
I think we should move the trouble-shooting section to BuildErrors and keep this doc focused on a build tutorial.